.github/workflows/PROJECT-PUBLISH-UPDATE-README.yaml

```
@@ -0,0 +1,72 @@
+name: UPDATE-PROJECT-README
+
+on:
+  workflow_dispatch:  # 수동 트리거 허용
+
+jobs:
+  publish-and-update:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)